### PR TITLE
Augment keyboard accessibility of the contacts menu

### PR DESCRIPTION
**Pull request by Pytal**
### To Do

- [x]  Fix focus border not visible in light mode
- [x]  Remove bottom border style erroneously applied to the elements within the contact
    - This was likely meant to remove the border for the last contact in the list but the border will be preserved to improve scannability
- [x]  Restore top contact action
  - Previous related changes can be found in https://github.com/nextcloud/server/pull/31931
- [x]  Navigate to and activate other actions with the keyboard

### Example

image

### CODE REVIEW

**Review by artonge — APPROVED** [review]

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
ah, actually this needs a rebase first @Pytal

**Review by artonge — CHANGES_REQUESTED** [review]

File: core/css/header.scss [hunk 1]
```diff
@@ -22,7 +22,7 @@
 	-moz-user-select: none;
 	-ms-user-select: none;
 	a:not(.button):focus-visible, button:not(.button-vue):focus-visible, div[role="button"]:focus-visible {
-		box-shadow: inset 0 0 0 2px var(--color-primary-text);
+		box-shadow: inset 0 0 0 2px var(--color-primary);
```
[the remark this answers is not in the document]
Comment (artonge) [reply core/css/header.scss:25]: If not, I did this one: https://github.com/nextcloud/server/pull/32742

[the remark this answers is not in the document]
Comment (Pytal) [reply core/css/header.scss]: Right the white outline is better, I've reverted the `header.scss` change and copied https://github.com/nextcloud/server/pull/32742/files#diff-38df8905a050af6ad588db71a76ddba1cae3f77e804073b8931c3e0fbaf731ec over here

**Review by artonge — APPROVED** [review]

**Pull request merged**